Development-Guidelines/Coding-Standards/naming-conventions.md

```diff
@@ -19,6 +19,6 @@
 ## C&#35;
 When developing new Class Libraries we will be adhering as closely as possible to the official guidelines as proposed by Microsoft [https://msdn.microsoft.com/en-us/library/ms229042.aspx](https://msdn.microsoft.com/en-us/library/ms229042.aspx)
 
-Another good reference is "Framework Design Guidelines: Conventions, Idioms, and Patterns for Reusable .NET Libraries" book by Krzysztof Cwalina and Brad Abrams
+Another good reference is[https://www.oreilly.com/library/view/framework-design-guidelines/9780321545671/] "Framework Design Guidelines: Conventions, Idioms, and Patterns for Reusable .NET Libraries" book by Krzysztof Cwalina and Brad Abrams.
```

Comment:
Thanks @dharmendra-lingaiah ! 😄 

Could you make sure it follow the MarkDown conventions that we use in the Documentation?
You can read more about it here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#external-links

Reply:
Done 👍